app/controllers/case_logs_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
 
     all_logs = current_user.case_logs
     unpaginated_filtered_logs = filtered_case_logs(filtered_collection(all_logs, search_term))
+    byte_order_mark = "\uFEFF"
```

Comment:
Wonder if this is worth DRY-ing? I assume we'd want it anywhere we download a csv?

Reply:
Good point, I've put byte_order_mark into application_controller instead :)